azure/packages/azure-client/package.json

```diff
@@ -65,12 +65,12 @@
 		"@fluidframework/driver-utils": "workspace:~",
 		"@fluidframework/fluid-static": "workspace:~",
 		"@fluidframework/map": "workspace:~",
-		"@fluidframework/protocol-definitions": "^3.0.0",
+		"@fluidframework/protocol-definitions": "^3.1.0-220363",
 		"@fluidframework/routerlicious-driver": "workspace:~",
 		"@fluidframework/runtime-utils": "workspace:~",
-		"@fluidframework/server-services-client": "^2.0.1",
+		"@fluidframework/server-services-client": "^3.0.0-220558",
```

Comment:
Since we're bumping to a new major server dep, this will want a changeset so consumers know that (came out of discussions with Loop). Doesn't need to be complicated, just point out that we are updating the deps, and if possible link to the release notes for server 3.0.

Reply:
the notes won't exist until a real (not-dev) release is done during the full release process.

Reply:
There should still be a change set with this commit. We (release drivers) will have to do manual work during the release to make sure all server changes are documented prior to the server release, which looks like will need to be today if this change is going in 8.0.

Reply:
@tylerbutler what should the change set say? the only change want set release tags in comments, but nothing that will impact an end user.